website/project/model.py

```diff
@@ -852,15 +855,43 @@ class Node(GuidStoredObject, AddonModelMixin, IdentifierMixin):
     # Primary institution node is attached to
     primary_institution = fields.ForeignField('institution')
 
+    affiliated_institutions = fields.ForeignField('institution', list=True)
+
     def add_primary_institution(self, user, inst):
         if inst.auth(user):
             self.primary_institution = inst
+            self.affiliated_institutions.append(inst)
```

Comment:
If you can, make this method idempotent, i.e. multiple invocations with the same input should be AOK.

```python
node.add_primary_institution(user, inst)  # adds institution
node.add_primary_instutition(user, inst)  # does not add instution--it is already added
```